src/components/manifold-subscription-list/manifold-subscription-list.tsx

```diff
@@ -64,6 +67,43 @@ export class ManifoldSubscriptionList {
     this.getSubscriptions(this.owner);
   }
 
+  ctaHref(planID: string) {
+    if (!this.baseUrl || this.baseUrl === '#') {
+      return this.baseUrl;
+    }
+
+    // const search = new URLSearchParams();
+    // set plan ID
+    // search.set('planId', planID);
+
+    // // set configurable feature selection (or skip, if no configurable features);
+    // Object.entries(this.userSelection[planID] || {}).forEach(([key, val]) => {
+    //   search.set(key, `${val}`);
+    // });
+
+    return `${this.baseUrl}/${planID.toString()}`;
```

Comment:
Hmm yeah, I am sensing that this is a bit awkward. @dangodev this is similar to something we did in UI, where we had a list of things and so we provided a URL _format_.  Is that an approach you'd suggest repeating or no?

Reply:
Yeah I’d recommend that here.

At least in the plan table component, we are using search params, which I’d also recommend here. So rather than `[base-url]/[plan-id]`, it should probably be `[base-url]?planId=[planId]` (and maybe other things after `planId` if needed)

Reply:
Otherwise they’d have to handle 1 route per plan, which would be a little annoying (especially because they’re non-human-readable)

Reply:
Well if we used formatting, wouldn't we not have to worry about query string vs. path at all? The consumer of the web component would make that decision. So they could easily specify something like,

```ts
const formatUrl = '/subscriptions?id=:subscriptionId&foo=bar';
```

We'd just be replacing `:subscriptionId` with the id of the item we clicked.